Client/Frontend/Browser/URLBarView.swift

```diff
@@ -148,6 +133,26 @@ class URLBarView: UIView, BrowserLocationViewDelegate, AutocompleteTextFieldDele
         finishEditingAnimation(false)
     }
 
+    private func makeTabsButton(count: Int) -> UIButton {
+        let tabsButton = InsetButton()
+        tabsButton.setTranslatesAutoresizingMaskIntoConstraints(false)
+        tabsButton.setTitle(count.description, forState: UIControlState.Normal)
+        tabsButton.setTitleColor(URLBarViewUX.BackgroundColor, forState: UIControlState.Normal)
+        tabsButton.accessibilityLabel = NSLocalizedString("Show Tabs", comment: "Accessibility Label for the tabs button in the browser toolbar")
+        tabsButton.titleLabel?.layer.backgroundColor = UIColor.whiteColor().CGColor
+        tabsButton.titleLabel?.layer.cornerRadius = 2
+        tabsButton.titleLabel?.font = AppConstants.DefaultSmallFontBold
+        tabsButton.titleLabel?.textAlignment = NSTextAlignment.Center
+        tabsButton.titleLabel?.snp_makeConstraints { make in
+            make.size.equalTo(URLBarViewUX.TabsButtonHeight)
+        }
+        tabsButton.addTarget(self, action: "SELdidClickAddTab", forControlEvents: UIControlEvents.TouchUpInside)
+        tabsButton.setContentHuggingPriority(1000, forAxis: UILayoutConstraintAxis.Horizontal)
```

Comment:
Nope, and apparently it's been discussed before in [this issue](https://github.com/SnapKit/Masonry/issues/88). I don't think think they have APIs for hugging and compression at all.